src/server/scripts/Northrend/UtgardeKeep/UtgardePinnacle/boss_svala.cpp

```diff
@@ -349,15 +346,19 @@ class boss_svala : public CreatureScript
                         case EVENT_RITUAL_PREPARATION:
                             if (Unit* sacrificeTarget = SelectTarget(SelectTargetMethod::Random, 0, 80.0f, true))
                             {
+                                me->InterruptNonMeleeSpells(true);
+                                me->SetReactState(REACT_PASSIVE);
+                                me->AttackStop();
+                                me->StopMoving();
                                 me->SetDisableGravity(true, true);
                                 instance->SetGuidData(DATA_SACRIFICED_PLAYER, sacrificeTarget->GetGUID());
                                 Talk(SAY_SACRIFICE_PLAYER);
                                 DoCast(sacrificeTarget, SPELL_RITUAL_PREPARATION);
-                                SetCombatMovement(false);
+                                DoCast(me, SPELL_RITUAL_DISARM);
```

Comment:
DoCastSelf pls for this and all other DoCast(me  scenarios

Reply:
thx ;)